src/test/java/teammates/test/cases/logic/FeedbackResponseCommentsLogicTest.java

```diff
@@ -85,19 +85,22 @@ public void testCreateFeedbackResponseComment() throws Exception {
         verifyPresentInDatastore(frComment);
 
         ______TS("typical successful case: frComment already exists");
+
+        FeedbackResponseCommentAttributes actualComment =
```

Comment:
I believe you are trying to verify that the comment exist in the database before the test.

```java
assertNotNull(frcLogic.getFeedbackResponseComment(frComment.getId()))
```

The above can already do the job?

Reply:
No, this is to fetch the existing comment from the database. I use this to verify that the duplicate comment has the same ID as the existing one.

Reply:
Then put it together with L97.

Reply:
L97? That would defeat the purpose, since after this I create a duplicate feedback response comment in the database. The variable `frcomment` will then be assigned the id. 

If I fetch `frcomment` from the database following that, it would always pass. I need to be able to fetch the original comment and verify that its ID is the same as the one given to `frcomment`

Unless, there is a better way to do this.

Reply:
> since after this I create a duplicate feedback response comment in the database.

Wait, we are not creating duplicate feedback response right? If the id is the same, the creation will be come update.

Reply:
Updated the documentation! Do see if its more clear now